src/config.ts

```diff
@@ -0,0 +1,7 @@
+const isProd = process.env.NODE_ENV === 'production';
```

Comment:
Sort of odd to me that we assume this is intended for replicache -- can a customer not have this defined in some other way that is important to them? In other words, should we be using a replicache-specific var eg env.REPLICACHE_ENV? (Not asserting the answer is yes, just asking.)

Reply:
This is a pretty strong convention in the JS community. 🤷‍♂️

Reply:
I'm not super happy with this but this is what React does